flake.nix

```diff
@@ -73,7 +73,7 @@
                             #!${pkgs.runtimeShell}
                             set -euo pipefail
 
-                            if [[ $DRY_ACTIVATE == "1" ]]
+                            if [[ "x''${DRY_ACTIVATE:-}" == "x1" ]]
```

Comment:
Why is `x` prepended here, are there cases where this won't work otherwise?

Reply:
You're right, doesn't make a difference.